Content.Client/GameObjects/Components/HUD/Inventory/ClientInventoryComponent.cs

```diff
@@ -91,6 +91,16 @@ public override void HandleComponentState(ComponentState curState, ComponentStat
                 doneSlots.Add(slot);
             }
 
+            if (cast.HoverEntity != null)
+            {
+                var slot = cast.HoverEntity.Value.Key;
+                var tuple = cast.HoverEntity.Value.Value;
+                var entity = Owner.EntityManager.GetEntity(tuple.Item1);
+                var fits = tuple.Item2;
```

Comment:
```suggestion
                var (slot, (entityUid, fits)) = cast.HoverEntity.Value;
                var entity = Owner.EntityManager.GetEntity(entityUid);
```

Can use deconstruction to make this much more concise now that you're using value tuples, by the way.